### PR TITLE
refactor(repo): refactor setup

### PR DESCRIPTION
**Pull request by Rubilmax**

### CODE REVIEW

File: test/transactions/helpers/MorphoAssociationTxTest.sol [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./TxTest.sol";
+
+/// @notice The DAO can call all the governance functions including the ones that can be used by Morpho Operator.
+/// It just needs to be executed through the Delay Modifier.
```
Comment (MerlinEgalite) [inline test/transactions/helpers/MorphoAssociationTxTest.sol:7]: This is true if I remember well, you must go through the delay for some specific functions like market creation, upgrade, etc. but not everything. I need to double check

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 Merlin Egalite
+Copyright (c) 2023 Morpho Labs
```
Comment (MerlinEgalite) [inline LICENSE:3]: lol wtf

**Comment by MerlinEgalite** [comment]
Huge work, thx!

File: test/transactions/helpers/MorphoAssociationTxTest.sol (hunk 1 above)
Comment (Rubilmax) [reply test/transactions/helpers/MorphoAssociationTxTest.sol:7]: Let's remove it then

**Review by MerlinEgalite — APPROVED** [review]

**Review by Jean-Grimal — APPROVED** [review]

**Pull request merged**